netlify/sitemap-index.xml

```diff
@@ -3,4 +3,13 @@
   <sitemap><loc>https://docs.projectcalico.org/release-legacy-sitemap.xml</loc></sitemap>
   <sitemap><loc>https://docs.projectcalico.org/latest-sitemap.xml</loc></sitemap>
   <sitemap><loc>https://docs.projectcalico.org/master/sitemap.xml</loc></sitemap>
+  <sitemap><loc>https://docs.projectcalico.org/archive/v3.20/sitemap.xml</loc></sitemap>
```

Comment:
I think it would be good to also add v3.21 to master. That way for the next release branch v3.21 is already there.